app/controllers/app/session_comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class App::SessionCommentsController < App::BaseController
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Not covered lines: 1 to 13